extensions/wikia/UserLogin/css/UserLogin.scss

```diff
@@ -75,3 +75,7 @@ $input-width: 230px;
 		display: none;
 	}
 }
+
+.hidden {
```

Comment:
@bkoval I would really like to do that :) However from my understanding, after talking with @lizlux there may be lots of cases where this rule gets overwritten with local rule with `!important` and it may contain slightly different rules.

If all @wikia-frontenders agree on making this happen, I would be super happy to take that on!